tests/core/test_batch.py

```diff
@@ -0,0 +1,223 @@
+from typing import Dict
```

Comment:
There were no tests for batch.py, and for all the optional arguments of `get_batch_request_from_acceptable_arguments`, the only one tested was `batch_request`. Now we have pretty full test coverage of this method.

All the tests in this file run in less than 1/20th of a second (in total)

Reply:
❤️